app-backend/migrations/src_migrations/main/scala/145.scala

```diff
@@ -0,0 +1,21 @@
+import slick.jdbc.PostgresProfile.api._
+import com.liyaos.forklift.slick.SqlMigration
+
+object M145 {
+  RFMigrations.migrations = RFMigrations.migrations :+ SqlMigration(145)(List(
+    sqlu"""
+ALTER TABLE aois ADD COLUMN shape UUID REFERENCES shapes(id);
```

Comment:
I would populate this column with shape IDs after migrating AOI area to shape geometry.